src/app/cube/details/details.component.ts

```diff
@@ -34,6 +34,10 @@ export interface Rating {
 export class DetailsComponent implements OnInit, OnDestroy {
   private isDestroyed$ = new Subject<void>();
   learningObject: LearningObject;
```

Comment:
Is there a way to not use this variable?  Currently there are 3 loaded objects at a time and all we need is 2 at most.

Reply:
There is actually only two loaded objects at most. The released is always loaded and if the user is a reviewer and there are revisions then the revised copy is loaded as well. The learningObject is what is currently displayed and is loaded with either the released or revised copies depending on the state of the toggle switch. I did this so the html didn't have if statements.